apps/emqx_dashboard_sso/src/emqx_dashboard_sso_ldap.erl

```diff
@@ -86,19 +86,7 @@ destroy(#{resource_id := ResourceId}) ->
 
 parse_config(Config0) ->
     Config = ensure_bind_password(Config0),
-    State = lists:foldl(
-        fun(Key, Acc) ->
-            case maps:find(Key, Config) of
-                {ok, Value} when is_binary(Value) ->
-                    Acc#{Key := erlang:binary_to_list(Value)};
-                _ ->
-                    Acc
-            end
-        end,
-        Config,
-        [query_timeout]
-    ),
-    {Config, State}.
+    {Config, emqx_ldap:parse_config(Config, [query_timeout], [])}.
```

Comment:
Q: should `query_timeout` be a binary ? prior to this change, it's a string.

Reply:
no, it is always an integer, in the prior code, it will never be convert to a string since it is not a binary